phone_bank_result.md

```diff
@@ -0,0 +1,804 @@
+                "created_date": "2017-03-20T21:04:31Z",
+                "modified_date": "2018-03-20T21:04:31Z",
+                "call_date": "2017-03-18T11:02:15Z",
+                "caller_id": "18552345678",
+                "phone_number": "15104024182",
+                "duration": 36,
+                "disposition": "answer",
+                "_links": {
+                    "self": {
+                        "href": "https://osdi-sample-system.org/api/v1/people/c945d6fe-929e-11e3-a2e9-12313d316c29/phone-bank-result/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bcf"
+                    },
+                    "osdi:caller": {
+                        "href": "https://osdi-sample-system.org/api/v1/people/c945d6fe-929e-11e3-a2e9-12313d316444"
+                    },
+                    "osdi:target": {
+                        "href": "https://osdi-sample-system.org/api/v1/people/c945d6fe-929e-11e3-a2e9-12313d316c29"
+                    },
+                    "osdi:answers": {
+                        "href": "https://osdi-sample-system.org/api/v1/people/c945d6fe-929e-11e3-a2e9-12313d316c29/phone-bank-result/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bcf/answers"
+                    },
+                    "osdi:taggings": {
+                        "href": "https://osdi-sample-system.org/api/v1/people/c945d6fe-929e-11e3-a2e9-12313d316c29/phone-bank-result/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bcf/taggings"
+                    },
+                    "osdi:attendances": {
+                        "href": "https://osdi-sample-system.org/api/v1/events/c945d6fe-929e-11e3-a2e9-12313d316c29/phone-bank-result/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bc3/attendances"
+                    },
+                    "osdi:phone_bank": {
+                        "href": "https://osdi-sample-system.org/api/v1/phone_bank/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0baa"
+                    }
+                }
+            },
+            //(truncated for brevity)
+        ]
+    }
+}
+```	
+
+_[Back to top...](#)_	
+
+### Scenario: Retrieving a collection of Phonebank result resources for a phone bank (GET)
+
+Calling this endpoint allows consumers to see a Phonebank's result history.
+
+#### Request
+
+```javascript
+GET https://osdi-sample-system.org/api/v1/api/v1/phone_bank/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0baa/phone-bank-result
+
+Header:
+OSDI-API-Token:[your api key here]
+```
+
+#### Response
+
+```javascript
+200 OK
+
+Content-Type: application/hal+json
+Cache-Control: max-age=0, private, must-revalidate
+
+{
+    "total_pages": 10,
+    "per_page": 25,
+    "page": 1,
+    "total_records": 250,
+    "_links": {
+        "next": {
+            "href": "https://osdi-sample-system.org/api/v1/phone_bank/d945d6fe-929e-11e3-a2e9-12313d316c29/phone-bank-result?page=2"
+        },
+        "osdi:phone-bank-results": [
+            {
+                "href": "https://osdi-sample-system.org/api/v1/people/c945d6fe-929e-11e3-a2e9-12313d316c29/phone-bank-result/d91b4b2e-ae0e-4cd3-9ed7-d0ec501b0bc3"
```

Comment:
shouldn't link to people here, right?

Reply:
this is also wrong on canvasses, should be fixed...

Reply:
@augfrank we are waiting on you to integrate Jason's comments, then we will approve